csrc/tensor_view.cpp

```diff
@@ -1372,50 +1391,39 @@ void TensorView::clearReductionIterDomains() {
   NVF_ERROR(
       !domain()->hasRoot(),
       "should not call clearReductionIterDomains on rfactor tv");
+  const std::vector<std::optional<bool>>& contiguity = getContiguity();
```

Comment:
This sounds like another use case for selfReplay. 
1. Create a TensorDomain of logical domain `noReductions(getLogicalDomain())`, loop empty, and allocation empty
2. selfReplay `this->domain()` to that new TensorDomain
3. setDomain(the new)

Am I missing something? Do I need to merge https://github.com/NVIDIA/Fuser/pull/5316 for selfReplay to work here?